op-chain-ops/genesis/config.go

```diff
@@ -290,8 +290,8 @@ type DeployConfig struct {
 	// UseInterop is a flag that indicates if the system is using interop
 	UseInterop bool `json:"useInterop,omitempty"`
 	// l2 blob related configs
```

Comment:
The comments should be updated to follow the existing config comments (e.g., `L2GenesisInteropTimeOffset`, etc).

Reply:
Done.